zntrack/zn/__init__.py

```diff
@@ -153,27 +152,31 @@ def load(self, instance):
 
 
 class outs(ZnTrackOption):
```

Comment:
Same sort of thing here. Of course it is not necessary to describe an option here but the ch class is seemingly quite specific and therefore I would argue requires a doc string.

Also, what is the idea behind the lowercase class names?

Reply:
see other comment